docs/getting_started/basics/ui_overview.md

```diff
@@ -1,63 +1,124 @@
 
 ## The Shift Window
 
-When you open Shift you will be prompted to a window like this:
-
-![Shift Window](../images/ui.png)
-
-Here you have all the components that will be used to create, test and debug your workflows.  
-
+<figure>
+      <img src="images/ui.png" alt="UI">
+      <figcaption><b>Figure 1</b>: Shift UI at startup.</figcaption>
+</figure>
+
+## Menu Bar
+At the very top of the application we find Shift's menu bar. This menu has all the actions that can be done in Shift. Most of them have short-cut associated.
+
+|Action|Menu|Description|Short-Cut|
+|:--|:--|:----|:--
+|*New Workflow*|*File*|Opens a new empty board.|`Ctrl+N`|
+|*Open Workflow*|*File*|Opens a file browser to open an existing workflow.|`Ctrl+O`|
+|*Open Recent*|*File*|Lists the most recent workflows to open.||
+|*Save Workflow*|*File*|Saves the current workflow.|`Ctrl+S`|
+|*Save Workflow As*|*File*|Saves the current workflow with a new name.|`Ctrl+Shift+S`|
+|*Catalog Manager*|*File*|Opens the [Catalog Manager](../../reference/catalogs#the-catalog-manager).|`P`|
+||||
+|*Undo*|*Edit*|Undoes the last action performed on the board.|`Ctrl+Z`|
+|*Redo*|*Edit*|Redoes the last undo performed on the board.|`Ctrl+Y`|
+|*Add New Node*|*Edit*|Prompts the "[Create Node](authoring#creating-a-node)" dialog.|`Ctrl+A`|
+|*Duplicate*|*Edit*|Duplicates the selected items.|`Ctrl+D`|
+|*Copy*|*Edit*|Copies the selected items to the clipboard.|`Ctrl+C`|
+|*Paste*|*Edit*|Pastes the items copied on the clipboard.|`Ctrl+V`|
+|*Cut*|*Edit*|Copies and removes the selected items to the clipboard.|`Ctrl+X`|
+|*Remove*|*Edit*|Removes the selected items from the board.|`Del`|
+|*Add Note*|*Edit*|Adds a new note to the board.|`N`|
+|*Add Plug*|*Edit*|If a node is selected, prompts the "[Add Plug](authoring#creating-new-plugs)" dialog.|`Ctrl+P`|
+||||
+|*Execute All*|*Execute*|Executes all the nodes in the workflow.|`Ctrl+Return`|
+|*Execute Selected*|*Execute*|Executes the selected nodes in the workflow.|`Shift+Return`|
+|*Execute Next*|*Execute*|Executes the next node in the workflow.|`>`|
+|*Clear Execution*|*Execute*|Clears the executed status from all the nodes in the workflow.|`<`|
+|*Stop On Error*|*Execute*|Defines if the workflow execution should stop when encountering an error or not.||
+||||
+|*Frame All*|*View*|Frames all the nodes present in the board.|`A`|
+|*Frame Selected*|*View*|Frames the selected nodes.|`F`|
+|*Frame Errors*|*View*|Frames all nodes that have errored.|`E`|
+|*Focus Execution*|*View*|Defines if the board should focus on the current executing node or not.||
+|*Log Level*|*View*|Defines the Logger's message log level.||
+||||
+|*About Shift*|*Help*|Opens an informative window about Shift.||
+
+Additional menus from the ones listed are plugin menus, which allow to extend Shift's functionalities and utility widgets. Please, refer to this [page](../../reference/plugins.md) for more information.
```

Comment:
```suggestion
Additional menus from the ones listed are Plugin menus, which allow to extend Shift's functionalities and utility widgets. Please, refer to this [page](../../reference/plugins.md) for more information.
```

Reply:
Because in this conversation, I asked how to write that word, because in different parts it was written in different ways: https://github.com/Inbibo/shift_docs/pull/37#discussion_r1726878160

Maybe we can agree to use it not capitalised and only capitalise it on specific cases if necessary 🤔

Reply:
Hmm I think we should treat all Shift concepts the same. Capitalizing all might be a bit too much, so I would rather leave it lower case.
@Inbibo/shift, what do you think?